Quantum-Control-Applications/ESR/Use case 1 - Sekhar Lab - CPMG/README.md

```diff
@@ -0,0 +1,102 @@
+
+_author: Kevin Villegas Rosales_
+
+_Demonstrated in the Lab of Prof. Sekhar at Darmouth College https://sites.dartmouth.edu/quantum-spin-lab/._
+
+_Demonstrated on the experiment of Ethan Williams. After two days of unboxing the OPX we
+were able to obtain the results listed below._
+
+_Important note: The code in this folder is the exact code that we used
+to run the experiment, so it is tailored for the specific software environment
+and set up. Thus, this is code is for inspirational purposes._
+
+## The goal
+
+To obtain the decoherence time of an ensemble of NV centers with CPMG dynamical decoupling
+sequence. Thus, it is a starting point for performing future noise spectroscopy experiments.
+
+## The device
+
+The device is a crystal containing an NV ensemble. The crystal is embedded in a room
+temperature ESR setup with a resonator at 2.5 GHz and 13 mT applied magnetic field.
+Note that the amplitude of the echo signal was obtained during the run-time
+of the experiment thanks for the demodulation capabilities of the OPX. Thus, there
+was no need for post-processing and the results shown below were obtained immediately
+after the end of the measurement.
+
+## Methods and results
+
+### CPMG sequence
+
+CPMG is a dynamical decoupling sequence that consist of a train of pulses. The effect of this
+train of pulses is to isolate the NV ensemble from noise at certain frequencies. Thus, it can
+serve to extend the lifetime of information encoded in qubits. A CPMG sequence can also be used to
+probe the noise environment surrounding the spins [1]. The image below displays the
+train of pulses applied in this experiment. After the hyperpolarization of spins, an initial
+`X_pi/2` pulse is sent and it is followed by a train of `Y_pi`. The delay between two consecutive
+`Y_pi` pulses is `2*tau`. In ESR experiments the echo amplitude is detected at `tau` time after
+`Y_pi` pulses.
+
+![sequence](CPMG_seq.png)
+
+### Echo amplitude decay in CPMG
+
+The measured decay traces with CPMG are shown below. The `y-axis` list the echo amplitude as demodulated
+and averaged in the OPX as the sequence evolves for `10 ms`. The `x-axis` is the `Time of echo` which
+is the time from the beginning of the sequence at which an *echo* was measured. The legend list all
+the different values of $\tau$ that were used. The traces displayed were obtained with only ten
+averages and in the timespan of a few minutes. The data transferred from the OPX to the computer was
+the data visualized in the figure and no post-processing was needed.
+
+We fit the experimental data with a double exponential following the equation below. For the rest of
+the analysis we focus on the measured $T_{2}$. A discussion about the `CPMG tail` can be found in
+Ref. [2]. The relative contribution of $T_{2}$ is at least three times larger than the `CPMG tail`.
+
+$$
+\begin{align}
+Echo\;amplitude &= A\exp(-t/T_{2}) + B\exp(-t/T_{tail})
```

Comment:
Should the T_2 have a power?

Reply:
I fixed to be like this like in their literature